### PR TITLE
fix(connect): Minor edits to Connect docs

### PR DESCRIPTION
**Pull request by lobsterkatie**
A few small fixes:

- remove SDK version so it's not immediately out of date
- break up installation code blocks so they both show (rather than being two tabs)
- space out options to make them easier to read and clarify option descriptions
- other random nits

### CODE REVIEW

File: src/platforms/node/connect.mdx [hunk 1]
```diff
@@ -9,11 +9,12 @@ If you are using `yarn` or `npm`, you can add our package as a dependency:
 
 ```bash
 # Using yarn
-$ yarn add @sentry/node@5.20.1
+$ yarn add @sentry/node@5.21.4
```
Comment (AbhiPrasad) [inline src/platforms/node/connect.mdx:12]: I don’t think we should specify version numbers to make sure we don’t have to keep updating this

Comment (lobsterkatie) [reply src/platforms/node/connect.mdx:12]: Fair. And agreed. It felt like more than I was going to do in the GH UI, LOL... but I suppose there's no reason we couldn’t just take the version off entirely. Will do that.

Comment (lobsterkatie) [inline src/platforms/node/connect.mdx:12]:
```suggestion
$ yarn add @sentry/node
```

File: src/platforms/node/connect.mdx [hunk 2]
```diff
@@ -9,11 +9,12 @@ If you are using `yarn` or `npm`, you can add our package as a dependency:
...
 
 # Using npm
-$ npm install @sentry/node@5.20.1
+$ npm install @sentry/node@5.21.4
```
Comment (lobsterkatie) [inline src/platforms/node/connect.mdx:15]:
```suggestion
$ npm install @sentry/node
```

**Pull request merged**